themes/V3/5ePHB/snippets.js

```diff
@@ -21,9 +21,34 @@ module.exports = [
 		view      : 'text',
 		snippets  : [
 			{
-				name : 'Table of Contents',
-				icon : 'fas fa-book',
-				gen  : TableOfContentsGen
+				name        : 'Table of Contents',
+				icon        : 'fas fa-book',
+				gen         : TableOfContentsGen,
+				subsnippets : [
+					{
+						name : 'Table of Contents',
+						icon : 'fas fa-book',
+						gen  : TableOfContentsGen,
+					},
+					{
+						name : 'Include H4',
+						icon : 'fas fa-dice-four',
+						gen  : dedent `{{tocH4
+							}}`,
+					},
+					{
+						name : 'Include H4, H5',
+						icon : 'fas fa-dice-five',
+						gen  : dedent `{{tocH5
+							}}`,
+					},
+					{
+						name : 'Include H4-H6',
+						icon : 'fas fa-dice-six',
+						gen  : dedent `{{tocH6
+							}}`,
+					},
```

Comment:
I figured it out eventually, but it was not obvious to me how these are meant to be used. Users will probably need more help. My thoughts along these lines:

1) We should probably include the `addToC` class to this list of snippets.
2) Adding `toH6` (or any of these) directly to a default-excluded element (monster stat block, for example) is a little weird, because it will include *only* the h4-h6 headers from that element. Perhaps these snippets by default should also behave like `addToC` (or just directly include `addToC` in the snippet), because if you are adding H4-H6, you probably want to include H1-H3 too.
3) Given 3, it might be more clear to users if we rename these:

> Include block in Table of Contents
> `{{addToC }}`

> Include up to H4.
> `{{addToC,tocH4 }}`

> Include up to H5.
> `{{addToC,tocH5 }}`

> Include up to H6.
> `{{addToC,tocH6 }}`

Not sure... Thoughts? @dbolack-ab @ericscheid

Reply:
Honestly, I had a hard time thinking about how to label these and label them concisely.  For the classes, the notion was that the # in tocH# was the lowest included header level.

These look good to me. @ericscheid ?